packages/@cruise-automation/worker-rpc/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@cruise-automation/worker-rpc",
```

Comment:
I wonder if we should just call it `@cruise-automation/rpc` to allow for using it over non-WebWorker channels in the future (e.g. websockets, like we discussed, or even things like [`BroadcastChannel`](https://developer.mozilla.org/en-US/docs/Web/API/Broadcast_Channel_API) to communicate between webworkers, or different Node processes, etc)

Reply:
renamed to `@cruise-automation/rpc`